src/processors/AsyncAPIInputProcessor.ts

```diff
@@ -63,7 +63,7 @@ export class AsyncAPIInputProcessor extends AbstractInputProcessor {
     
     let schemaUid = schema.uid();
     //Because the constraint functionality of generators cannot handle -, <, >, we remove them from the id if it's an anonymous schema.
-    if (schemaUid.includes('<anonymous-schema')) {
+    if (typeof schemaUid !== 'undefined' && schemaUid.includes('<anonymous-schema')) {
```

Comment:
I'm not confident about this revision, since it's to the AsyncAPI input processor used by all generators (not just the Rust generator). It'd help to see the input you're using. Maybe @jonaslagoni has better insight here.

Reply:
I checked more. The issue seems from AsyncAPI side (not specific to Rust).

If one of the schemas in the spec is missing `type`, this error happens:

```
/Users/yushi.omote/dev/model-play/modelina/src/processors/AsyncAPIInputProcessor.ts:66
    if (schemaUid.includes('<anonymous-schema')) {
                  ^
TypeError: Cannot read properties of undefined (reading 'includes')
```

This is the minimal spec that reproduces the issue:

```yaml
asyncapi: 2.0.0
id: "https://github.com/yushiomote/hoge"
info:
  title: "Test"
  version: 2.0.0
defaultContentType: application/json
channels:
  hello:
    publish:
      message:
        name: hello
        payload:
          # type: object <- The issue happens if this is missing
          properties:
            data: { type: string }
```

Maybe we should define `type` (it's supposed to be `required` field?) but Asyncapi studio doesn't blame the missing type
<img width="1390" alt="Screen Shot 2022-09-11 at 14 42 46" src="https://user-images.githubusercontent.com/3733915/189514505-49edb9da-8651-47fb-b95a-7ff5f4405975.png">

Reply:
I can move this specific revision to a different PR opening a separate issue if needed 🙇

Reply:
As this is not related rust specific problems, I think it makes sense to split it out if you can @YushiOMOTE 🙂

Reply:
Solving in https://github.com/asyncapi/modelina/pull/876